starboard/linux/shared/time_zone.cc

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "starboard/linux/shared/time_zone.h"
+
+#include "starboard/extension/time_zone.h"
+
+#include <stdlib.h>
+#include <time.h>
+
+namespace starboard {
+namespace shared {
+
+namespace {
+
+// Definitions of any functions included as components in the extension
+// are added here.
+
+void SetTimeZone(const char* time_zone_name) {
+  setenv("TZ", time_zone_name, 1);
```

Comment:
Should we check for null?

Reply:
If it's null, then the TimeZone will be set as the default value. Based on my understanding, we will only use this in our nplb tests, and we will use it the way that it's never null? Though, it doesn't hurt to handle the case. If it's null we just make the function to early return?